app/views/shared/_comment.html.erb

```diff
@@ -1,7 +1,7 @@
-<% commentable ||= comment.commentable %>
+<% commentable ||= comment.commentable.becomes(comment.commentable.class.base_class) %>
 <div class="comment-box">
   <div class="comment">
-    <h5 class="comment-title"><%= comment.title %></h5>
+    <h5 class="comment-title"><%= comment.title %> <%= badge(commentable.class.name, style: 'info') %></h5>
     <div class="comment-description"><%= comment.description %></div>
```

Comment:
Because requests can contain comments for the labware/receptacles I just added a little badge to help determine where the comment came from.